modules/services/rsibreak.nix

```
@@ -0,0 +1,38 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.rsibreak;
```

For consistency with other packages:
```suggestion
let

  cfg = config.services.rsibreak;
```